src/main/drivers/mcu/apm32/camera_control.c

```diff
@@ -0,0 +1,268 @@
+/*
```

Comment:
`camera_control.c` has to be split to common and mcu specific part first.

This leads to huge code duplication.